content/departments/product-engineering/engineering/enablement/repo-management/support-new-code-host.md

```diff
@@ -0,0 +1,52 @@
+# Adding support for a new code host
+
+Sourcegraph currently supports a number of different [code hosts](https://docs.sourcegraph.com/admin/external_service).
+
+Not all code hosts are supported "natively". What we mean by this is that the code host type is listed an an option in our UI and it seamless to add. In some cases it is possible to work around this if the code host is based on `git`, in which case it can be added via our [other git repository hosts](https://docs.sourcegraph.com/admin/external_service/other) mechanism.
```

Comment:
```suggestion
Not all code hosts are supported "natively". Native supports means a code host type is listed as an option in our UI and it is seamless to add. In some cases it is possible to work around this if the code host is based on `git`, in which case it can be added via our [other git repository hosts](https://docs.sourcegraph.com/admin/external_service/other) mechanism.
```